Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Constellar Minors 2

## Changelog:
- Added Koltira and Faltora Brightweavers.
- Added Thassarian and his family.
- Added Hekazi and Zekhan (AKA Zappy Boi).
- Fizzle is no longer subservient to Gnomeregan as his affiliation wasn't cemented before the events of Cataclysm (lore-wise IG year 610-611).

## How to test:
Check whether the would-be-death knight characters appear properly in their races' courts (or Arthas' post-603). Also confirm they are friends in 605 startdate.
Check whether Hekazi and/or Zekhan appear properly in Sen'jin / Vol'jin's court (depending on startdate).
Check if Fizzle's holdings are properly independent from the Kingdom of Gnomeregan in 605 startdate.
